microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -83,10 +83,8 @@ class AzureImageStandard(TestSuite):
     # [WARNING]: Running ['tdnf', '-y', 'upgrade'] resulted in stderr output.
     # cloud-init[958]: photon.py[ERROR]: Error while installing packages
     _ERROR_WARNING_pattern: List[Pattern[str]] = [
-        re.compile(r"^(.*\[ERROR\]*)$", re.MULTILINE),
-        re.compile(r"^(.*ERROR:*)$", re.MULTILINE),
-        re.compile(r"^(.*\[WARNING\]*)$", re.MULTILINE),
-        re.compile(r"^(.*WARNING:*)$", re.MULTILINE),
+        re.compile(r"^(.*[ERROR]*)$", re.MULTILINE),
+        re.compile(r"^(.*[WARNING]*)$", re.MULTILINE),
```

Comment:
The new pattern looks incorrect and wide. It matches like `W`, `AWRING`, `IN` or something else.

Reply:
Please add newly examples above, so you can test the new patterns without regressions.

Reply:
Thanks for pointing this out. I tested the new pattern ^(.*\[WARNING\]:*) on https://regex101.com/. Also, I removed the $ sign to make the match more tolerate. For now, the only pattern strings in cloud-init.log we want to match are "[ERROR]:" and "[WARNING]:"